github/issues.go

```diff
@@ -47,6 +47,7 @@ type Issue struct {
 	// TextMatches is only populated from search results that request text matches
 	// See: search.go and https://developer.github.com/v3/search/#text-match-metadata
 	TextMatches []TextMatch `json:"text_matches,omitempty"`
+	Locked      *bool       `json:"locked,omitempty"`
```

Comment:
Please move it up, together with all the other core fields.

This makes it look like its grouped with `TextMatches`, but it isn't.

You can place it after `Assignees`, or somewhere else if you think it's more relevant/related to another existing field.

Perhaps under `State` is a good fit? Or above `Comments`, since that's where it shows up in examples on GitHub API docs.